src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -83,10 +91,6 @@ private function configureRoutes(RoutingConfigurator $routes): void
         } else {
             $routes->import($configDir.'/{routes}.php');
         }
-
-        if (false !== ($fileName = (new \ReflectionObject($this))->getFileName())) {
-            $routes->import($fileName, 'attribute');
```

Comment:
move these lines above changes the precedence of route definitions, not sure this is desired
can be reverted after changing getConfigDir